lib/vectorsearch/weaviate.rb

```diff
@@ -63,14 +58,9 @@ def create_default_schema
     # @param k [Integer|String] The number of results to return
     # @return [Hash] The search results
     def similarity_search(query:, k: 4)
-      near_text = "{ concepts: [\"#{query}\"] }"
+      embedding = generate_embedding(text: query)
```

Comment:
Right now we just use `llm_client.embed`. `generate_embedding` alias was removed.